rosdistro_snapshot.yaml

```diff
@@ -3207,6 +3207,10 @@ mola_yaml:
   tag: release/jazzy/mola_yaml/2.1.0-1
   url: https://github.com/ros2-gbp/mola-release.git
   version: 2.1.0
+mocap4r2_msgs:
+  tag: release/jazzy/mocap4r2_msgs/0.1.0-1
+  url: https://github.com/MOCAP4ROS2-Project/mocap4r2_msgs-release.git
+  version: 0.1.0
```

Comment:
In theory we should add this to https://github.com/RoboStack/ros-jazzy/blob/main/rosdistro_additional_recipes.yaml as this file is actually automatically generated. Anyhow, it is probably a good idea to add this here, so it will be automatically substituted with the new version on a full rebuild that refresh the snapshot.